### PR TITLE
Update lifecycle

### PR DESCRIPTION
**Pull request by TravisEz13**
Version(s) of document impacted
------------------------------
- [x] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by TravisEz13** [comment]
@sdwheeler I'll deal with conflicts after you merge anything pending.  Just ping me when I need to resolve conflicts.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[181f99b](https://github.com/TravisEz13/PowerShell-Docs/pull/4074/commits/181f99b31380800fb42ee465880a595140f74a2d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/PowerShell-Support-Lifecycle.md](https://github.com/TravisEz13/PowerShell-Docs/blob/update_lifecyle/reference/docs-conceptual/PowerShell-Support-Lifecycle.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-3.0&branch=pr-en-us-4074) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-4.0&branch=pr-en-us-4074) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-5.0&branch=pr-en-us-4074) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-5.1&branch=pr-en-us-4074) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-6&branch=pr-en-us-4074) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903282317174845-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fHKYAL3ylu%2BjIx8WGlaLFuIGG7PLyK1ZUKEiUe%2B8GA4%3D&st=2019-03-28T23%3A16%3A59Z&se=2019-04-28T23%3A21%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/PowerShell-Support-Lifecycle.md [hunk 1]
```diff
@@ -113,6 +111,7 @@ So, the distribution owners ended support for the following versions and are not
 | Ubuntu   | 17.10   | [July 2018](https://lists.ubuntu.com/archives/ubuntu-announce/2018-July/000232.html)        |
 | Debian   | 8       | [June 2018](https://lists.debian.org/debian-security-announce/2018/msg00132.html)           |
 | Fedora   | 27      | [November 2018](https://fedoramagazine.org/fedora-27-end-of-life/)                          |
+| Ubuntu   | 14.04   | [April 2018](https://wiki.ubuntu.com/Releases)                                              |
```
Comment (DCtheGeek) [inline reference/docs-conceptual/PowerShell-Support-Lifecycle.md]: Shouldn't this be April 2019?

File: reference/docs-conceptual/PowerShell-Support-Lifecycle.md [hunk 2]
```diff
@@ -57,30 +57,27 @@ These packages are marked as `Community` in the table.
...
-|                                                   | 6.0         | 6.1         |
+|                                                   | 6.1         | 6.2         |
 |---------------------------------------------------|:-----------:|:-----------:|
 | Windows 7, 8.1, and 10                            | Supported   | Supported   |
 | Windows Server 2008 R2, 2012 R2, 2016             | Supported   | Supported   |
 | [Windows Server Semi-Annual Channel][semi-annual] | Supported   | Supported   |
-| Ubuntu 14.04 and, 16.04                           | Supported   | Supported   |
-| Ubuntu 18.04                                      |             | Supported   |
-| Ubuntu 18.10 (via Snap Package)                   |             | Community   |
+| Ubuntu 16.04 and, 18.04                           | Supported   | Supported   |
```
Comment (DCtheGeek) [inline reference/docs-conceptual/PowerShell-Support-Lifecycle.md]: Remove the comma

**Comment by TravisEz13** [comment]
Thanks @DCtheGeek

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[caf3845](https://github.com/TravisEz13/PowerShell-Docs/pull/4074/commits/caf3845aaa220a005a12b8aa94b8283857da72da)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/PowerShell-Support-Lifecycle.md](https://github.com/TravisEz13/PowerShell-Docs/blob/update_lifecyle/reference/docs-conceptual/PowerShell-Support-Lifecycle.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-3.0&branch=pr-en-us-4074) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-4.0&branch=pr-en-us-4074) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-5.0&branch=pr-en-us-4074) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-5.1&branch=pr-en-us-4074) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/powershell-support-lifecycle?view=powershell-6&branch=pr-en-us-4074) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C28%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903282351361068-4074%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JCSpsV1hrNegdfBpMDqCMv9DVBWGxQB0IyKvfL94atI%3D&st=2019-03-28T23%3A51%3A25Z&se=2019-04-28T23%3A56%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**